### PR TITLE
Add some entries (Fehn, Kronbichler, Munch, Schoeder)

### PR DESCRIPTION
**Pull request by peterrum**

### CODE REVIEW

File: publications-2019.bib [hunk 1]
```diff
@@ -678,4 +678,109 @@ @article{Fei2019
...
+@article{kronbichler2019fast,
+  title={Fast matrix-free evaluation of discontinuous Galerkin finite element operators},
+  author={Kronbichler, Martin and Kormann, Katharina},
+  journal={ACM Transactions on Mathematical Software (TOMS)},
+  volume={45},
+  number={3},
+  pages={1--40},
+  year={2019},
+  publisher={ACM New York, NY, USA}
+}
```
Comment (kronbichler) [inline publications-2019.bib]: I think you got this one twice - it is already in line 694.

File: publications-2019.bib [hunk 2]
```diff
@@ -678,4 +678,109 @@ @article{Fei2019
...
+@article{fehn2019hybrid,
+  title={Hybrid multigrid methods for high-order discontinuous Galerkin discretizations},
+  author={Fehn, Niklas and Munch, Peter and Wall, Wolfgang A and Kronbichler, Martin},
+  journal={arXiv preprint arXiv:1910.01900},
+  year={2019}
+}
+
+
+@article{fehn2019high,
+  title={High-order DG solvers for underresolved turbulent incompressible flows: A comparison of L 2 and H (div) methods},
```
Comment (kronbichler) [inline publications-2019.bib]:
```suggestion
  title={High-order DG solvers for underresolved turbulent incompressible flows: A comparison of L2 and H(div) methods},
```

File: publications-2019.bib [hunk 3]
```diff
@@ -678,4 +678,109 @@ @article{Fei2019
...
+  journal={International Journal for Numerical Methods in Fluids},
+  volume={91},
+  number={11},
+  pages={533--556},
+  year={2019},
+  publisher={Wiley Online Library}
+}
+
+
+@article{fehn2019modern,
```
Comment (kronbichler) [inline publications-2019.bib]: Can you remove the entry from the 2018 list?

File: publications-2019.bib [hunk 4]
```diff
@@ -678,4 +678,109 @@ @article{Fei2019
...
+  journal={International journal for numerical methods in biomedical engineering},
+  volume={35},
+  number={12},
+  pages={e3228},
+  year={2019},
+  publisher={Wiley Online Library}
+}
+
+
+@article{fehn2019matrix,
```
Comment (kronbichler) [inline publications-2019.bib]: Can you remove the respective entry from the 2018 list?

File: publications-2019.bib [hunk 5]
```diff
@@ -678,4 +678,109 @@ @article{Fei2019
...
+  journal={International Journal for Numerical Methods in Fluids},
+  volume={89},
+  number={3},
+  pages={71--102},
+  year={2019},
+  publisher={Wiley Online Library}
+}
+
+
+@PhdThesis {
```
Comment (kronbichler) [inline publications-2019.bib]: Choose some handle, e.g. `schoeder2019phd`

**Comment by kronbichler** [comment]
Would you mind fixing the build error (invalid character in the hybrid multigrid paper)?

**Comment by peterrum** [comment]
@kronbichler Ready!

**Comment by kronbichler** [comment]
Could you please rebase - no reason to keep the invalid character in the history. :laughing:

**Pull request merged**